scripts/relup-base-packages.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+if [[ -n "$DEBUG" ]]; then
+    set -x
+fi
+set -euo pipefail
+
+# ensure dir
+cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")/.."
+
+PROFILE="${1}"
+if [ "$PROFILE" = "" ]; then
+    PROFILE="emqx"
+fi
+
+OTP_VSN=${OTP_VSN:-$(scripts/get-otp-vsn.sh)}
+
+case $PROFILE in
+    "emqx")
+        DIR="broker"
+        ;;
+    "emqx-ee")
+        DIR="enterprise"
+        ;;
+    "emqx-edge")
+        DIR="edge"
+        ;;
+esac
+
+SYSTEM="$(./scripts/get-distro.sh)"
+
+ARCH="$(uname -m)"
+case "$ARCH" in
+    x86_64)
+        ARCH='amd64'
+        ;;
+    aarch64)
+        ARCH='arm64'
+        ;;
+    arm*)
+        ARCH=arm
+        ;;
+esac
+
+SHASUM="sha256sum"
+if [ "$SYSTEM" = "macos" ]; then
+    SHASUM="shasum -a 256"
+fi
+
+mkdir -p _upgrade_base
+pushd _upgrade_base
+
+for tag in $(../scripts/relup-base-vsns.sh community | xargs echo -n); do
```

Comment:
```suggestion
for tag in $(../scripts/relup-base-vsns.sh $PROFILE | xargs echo -n); do
```

Reply:
Updated, we have updated the `relup-base-vsns.sh` script ?

Reply:
no relup-base-vsns.sh is not changed,
5.0 enterprise edition has profile named `emqx-enterprise` matches the `*enterprise*` pattern.